src/interfaces/IBlockInfo.ts

```diff
@@ -6,7 +6,6 @@ export interface IBlockInfo {
   content: null | {
     is_final: boolean;
     is_stale: boolean;
-    is_in_blockclique: boolean;
```

Comment:
Can you share more information on why we need o remove it ?

Reply:
The is_in_blockClique appears twice : the first is in the "content" and the second one in the "root". I asked @AurelienFT if it was wanted and he told me that it was an "is_in_blockclique" located in the content was an error

Reply:
great thanks!